imagetagger/imagetagger/images/management/commands/runzipdaemon.py

```diff
@@ -0,0 +1,48 @@
+import fasteners
+import os
+import sys
+from time import sleep
+
+import zipfile
+from django.conf import settings
+from django.core.management import BaseCommand
+from django.db import transaction
+from django.db.models import Q
+
+from imagetagger.images.models import ImageSet
+
+
+class Command(BaseCommand):
+    help = 'Runs a daemon that creates zip archives for image sets and keeps them up to date'
+
+    def handle(self, *args, **options):
+        if not settings.ALLOW_ZIP_DOWNLOAD:
+            sys.stderr.write('To enable zip download, set ALLOW_ZIP_DOWNLOAD to True in settings.py\n')
```

Comment:
Uargh, at least use self.stderr instead of using sys directly. However, you should rather raise a CommandError instead.

(cf. https://docs.djangoproject.com/en/2.1/howto/custom-management-commands/)